gnome/gstreamer1-gst-plugins-good/Portfile

```diff
@@ -10,6 +10,7 @@ name                gstreamer1-gst-plugins-good
 set my_name         gst-plugins-good
 # please only commit stable updates (even numbered releases)
 version             1.16.1
+revision            1
```

Comment:
There is no need to change the `revision` here.

Reply:
OK, fixed in commit 2cd158c.